client/js/bar.js

```diff
@@ -46,11 +46,22 @@ function createBarChart(chartValues, chartCategories, maxChartVal) {
     bar.setAttribute('x', GRAPH_LEFT_PADDING);
     bar.setAttribute('y', graphTopPadding + barContainerHeight * i);
 
+    const value = document.createElementNS(SVG_NS, 'text');
+    g.append(value);
+    value.setAttribute('class', 'bar-value');
+    value.setAttribute('x', 3 * GRAPH_LEFT_PADDING);
+    value.setAttribute('y', barContainerHeight * i + barThickness);
+    value.setAttribute('text-anchor', 'middle');
+    value.fill = 'white';
+    value.textContent = chartValues[i];
```

Comment:
not sure if this is the best way to format within the svg number within the svg rects.
I do it by setting the coordinates. Other advice is appreciated!